src/components/CreatePostButon/index.jsx

```diff
@@ -0,0 +1,39 @@
+import { Button, BUTTON_OPTIONS } from '../Button';
+import { Heading, HEADING_OPTIONS } from '../Heading';
+import { NavLink } from 'react-router-dom';
+import React from 'react';
+
+const CreatePostButton = () => {
+  return (
+    <>
+      <NavLink exact to="/post">
+        <Button
+          shadow={true}
+          isRounded={true}
+          color={BUTTON_OPTIONS.BACKGROUND_COLOR.PrimaryBlack}
+          className="flex flex-shrink py-3 px-4 my-6"
+        >
+          <i className="mr-3 hidden sm:block h-8 w-8">
```

Comment:
- don't hide the Icon on the small screen
- the clickable area should be just the button, not the whole page div
- make the icon a little bit small, give it a (w-5 h-5) classes instead of 8
- and it would be better if you can find an icon with a grey color or less black